lib/client/player.go

```diff
@@ -215,7 +238,30 @@ func (p *sessionPlayer) playRange(from, to int) {
 		}
```

Comment:
looks like the write to `p.position` on the line above (gh won't let me comment there) is still unsynchronized

Reply:
I thought so too, but I couldn't get the race detector to trip on this as-is (probably because the code ensures you only ever have one playback routine running at a time).

Added a lock anyway.

Reply:
Yeah it actually looks like it would be implicitly protected, the lock can't hurt though